references/detection/train.py

```diff
@@ -176,6 +183,8 @@ def main(args):
 
     print("Creating model")
     kwargs = {"trainable_backbone_layers": args.trainable_backbone_layers}
+    if args.data_augmentation in ["multiscale", "lsj"]:
+        kwargs["_skip_resize"] = True
```

Comment:
Because Multi-scale and LSJ require to do the resizing outside the model, this is necessary. For now I don't expose the property publicly (we got other params like this), but this is something we should consider removing from the models. See #2263 for a similar request.